protocol/identity_images.go

```diff
@@ -77,7 +77,7 @@ image:
 			// Decrypt the main encryption AES key with AES encryption using the DH key
 			dAESKey, err := common.Decrypt(empk, sharedKey)
 			if err != nil {
-				if err.Error() == "cipher: message authentication failed" {
+				if err.Error() == "cipher: message authentication failed" { // nolint: goconst
```

Comment:
we should probably const this instead of disabling lint :)

Reply:
fixed :)